drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -443,8 +441,63 @@ function DragAndDropTemplates(configuration) {
         )
     };
 
+    var progressTemplate = function(ctx) {
+        // Formats a number to 4 decimals without trailing zeros
+        // (1.00 -> '1'; 1.50 -> '1.5'; 1.333333333 -> '1.3333').
+        var formatNumber = function(n) { return n.toFixed(4).replace(/\.?0+$/, ''); };
+        var is_graded = ctx.graded && ctx.max_score > 0;
```

Comment:
Suggestion: Consider having more descriptive variable names in this code so the reader can easily distinguish between `weighted_max_score` versus `raw_max_score`, for example.
(We did a similar renaming exercise recently in the grading infrastructure in the platform for readability and to prevent bugs.)